README.md

```diff
@@ -68,6 +70,7 @@ After writing this method, our new result should be:
 ```
 
 **think**: Where should we call `strip_punctuation`? Does it matter? Why?
+We should call strip_punctuation before running the analyze_mood. It matters because analyze_mood cannot produce an accurate result until the punctuation is removed.
```

Comment:
:+1:  

But do you think you should call strip_punctuation inside or outside the analyze_mood method? Why?